src/globals.js

```diff
@@ -112,3 +112,25 @@ export const StringMap = typeof g.Map === 'function' &&
       });
     }
   };
+
+export const StringSet = g.Set || function (input) {
```

Comment:
When landing, I'll change this one to a `typeof` ternary to match `g.Map`. I tend to not bother adding a type checks for most polyfills (e.g. Array or String prototype methods) as I agree that presence is sufficient. If it's set to something else, that's asking for trouble!

But, global constructors in particular are prone to conflicting with unrelated values. E.g. `<h2 id="Set">Set</h2>` in an HTML fixture will result in `g.Set` coming through the `window` object via WindowProxy as HTMLElement reference. It's one of those legacy features on the web you are likely to run into sooner or later even if you're not (intentionally) looking for it.

<https://html.spec.whatwg.org/#named-access-on-the-window-object>